src/lib.rs

```diff
@@ -15,8 +73,23 @@ use std::fs::{self, File, OpenOptions};
 use std::io::{Error as IoError, ErrorKind::NotFound, Write};
 use std::path::PathBuf;
 
-/// Load a configuration from the standard OS local scope for
-/// the current user.
+/// Load an application configuration from disk
+///
+/// A new configuration file is created with default values if none
+/// exists.
+///
+/// Errors that are returned from this function are IO related,
```

Comment:
"I/O"

unrelated to this PR: We should decorate the errors to include some context. this would change them to no longer be `io::Error`s